src/portoctl.cpp

```diff
@@ -376,25 +378,25 @@ class TSetPropertyCmd : public ICmd {
     }
 };
 
-class TGetDataCmd : public ICmd {
+class TGetDataCmd final : public ICmd {
 public:
     TGetDataCmd(TPortoAPI *api) : ICmd(api, "dget", 2, "[-k] <container> <data> [data...]", "get raw container data") {}
 
-    int Execute(int argc, char *argv[]) {
+    int Execute(TCommandEnviroment *env) final override {
         bool printKey = false;
-        int start = GetOpt(argc, argv, {
+        const auto &args = env->GetOpts({
```

Comment:
Unfortunately, I was unable to move these static defined options to class members (
It uses closures with local vars and class-scope vars and such change will lead to massive commands rewrite.